### PR TITLE
Tenant Schema

### PR DESCRIPTION
**Pull request by dvaldivia**
Screen Shot 2019-10-21 at 1 57 02 PM

### CODE REVIEW

**Review by cesnietor — COMMENTED** [review]
Based on yesterdays discussion, this is outdated.
Groups will not be there anymore.

**Comment by dvaldivia** [comment]
Just updated it

**Comment by dvaldivia** [comment]
The schema looks like this now:

Screen Shot 2019-10-22 at 4 23 09 PM

File: cluster/tenant-migrations/000001_tenant-initial.up.sql [hunk 1]
```diff
@@ -0,0 +1,114 @@
+create table atenantschema.users
```
Comment (cesnietor) [inline cluster/tenant-migrations/000001_tenant-initial.up.sql]: is `atenantschema` misspelled? or is it supposed to be like that?
Also where is this schema created?

**Review by cesnietor — APPROVED** [review]

File: cluster/tenant-migrations/000001_tenant-initial.up.sql (hunk 1 above)
Comment (dvaldivia) [reply cluster/tenant-migrations/000001_tenant-initial.up.sql]: it was never meant to be, I designed the tables on that schema, should have not been checked in

**Pull request merged**